scripts/presubmit.sh

```diff
@@ -126,7 +127,11 @@ main() {
 
       # Do not run go test in the loop, instead echo it so we can use xargs to
       # add some parallelism.
-      echo go test -short -timeout=5m ${coverflags} ${goflags} "$d"
+      echo go test \
+          -short \
+          -timeout=${GO_TEST_TIMEOUT:=5m} \
```

Comment:
unsure of this bash syntax.  is the = meant to be there (I assume this marks the default when GO_TEST_TIMEOUT isn't set?)

Reply:
The `:=` means if `var` is set to null or undeclared then set it to `5m` and use that value.
More frequently you see `:-` which means if... null or undeclared then just use this default value.

Reply:
Yes, `:=` defines a default value in case the var is not set.

I forget those frequently too, so my trick is to lookup "expansion" on [man bash](https://linux.die.net/man/1/bash) (search for "assign default value" for an easier time).

Reply:
@AlCutter, oh, right, I guess I meant to use `:-` instead of `:=`.

*codingllama silently despairs about the prospect of facing yet another Travis run*